environment/interpretation/lane_detector/data/Samples.model

```diff
@@ -1,42 +1,35 @@
-																											svm_type c_svc
+svm_type c_svc
```

Comment:
@aranyadan Can you please comment about why these additions were made. Put it as comments in the code.